server/enums/challengeCategory.ts

```diff
@@ -1,11 +1,11 @@
 enum ChallengeCategory {
   SENSORY = 'SENSORY',
-  EMOTIONS_FEELINGS_DEFAULT = 'EMOTIONS_FEELINGS_DEFAULT',
```

Comment:
All of these original "category" values were actually "types", in that they are the default type codes for challenges

The new values are the actual categories 👍